### DISCUSSION TITLE
Maj de version

### DISCUSSION THREAD

**Discussion by Francisco-Ordas** [dbarzin/mercator#493] [Q&A]
Bonjour
Novice dans Mercator, j'ai installé deux instances sur deux 2 VM distinctes à quelques semaines d'intervalle.
Je constate que les deux instances ne présentent pas les mêmes objets, exemple la vue RGPD et d'autres choses comme le paramétrage d'une Entité externe à l'organisme...
Je suppose qu'elles ne sont pas dans la même version ?
Si oui, comment vérifier la version et comment mettre à jour vers la version la plus récente ?
Merci
Francisco

**Comment by dbarzin** [comment]
La procédure de mise à jour est documentée ici : https://github.com/dbarzin/mercator/blob/master/INSTALL.fr.md
Il n'y a pas de numéro de version présent dans l'application. C'est dans la feuille de route.
